packages/@okta/vuepress-site/docs/release-notes/2020-07-0/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2020.07.0
+
+| Change                                                                                                                                                      | Expected in Preview Orgs |
+| ----------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ |
+| [Apple as an Identity Provider is now GA in Preview](#apple-as-an-identity-provider-is-now-ga-in-preview)                                                   | July 9, 2020             |
+| [YubiKey OTP Token operations added](#yubikey-otp-token-operations-added)                                                                                   | July 9, 2020             |
+| [Support for creating OIN OIDC Apps via the Dynamic Client Registration API](#support-for-creating-oin-oidc-apps-via-the-dynamic-client-registration-api)   | July 9, 2020             |
+| [API support for multiple ACS URLs](#api-support-for-multiple-acs-urls)                                                                                     | July 9, 2020             |
+| [Bugs fixed in 2020.07.0](#bugs-fixed-in-2020-07-0)                                                                                                         | July 9, 2020             |
+
+### Apple as an Identity Provider is now GA in Preview
+
+Apple as an Identity Provider is now Generally Available in Preview. Apple as an IdP allows users to sign in to your app using their Apple ID. See [Apple as an Identity Provider](/docs/guides/add-an-external-idp/apple/before-you-begin/). <!-- OKTA-302300 -->
+
+### YubiKey OTP Token operations added
+
+Using the [Factors API](/docs/reference/api/factors/), requests for single YubiKey OTP Tokens, and uploading a seed for a YubiKey OTP are now supported. Other API operations for YubiKey OTP Tokens are now documented in the [Factors API](/docs/reference/api/factors/). <!-- OKTA-302434 -->
+
+### Support for creating OIN OIDC Apps via the Dynamic Client Registration API
```

Comment:
via -> through

style is to avoid the use of "via" and use "through" instead or "using" in this situation?